forgerock-ui-commons/pom.xml

```diff
@@ -1,44 +1,45 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-  ~ DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
-  ~
-  ~ Copyright (c) 2011-2016 ForgeRock AS. All Rights Reserved
-  ~
-  ~ The contents of this file are subject to the terms
-  ~ of the Common Development and Distribution License
-  ~ (the License). You may not use this file except in
-  ~ compliance with the License.
-  ~
-  ~ You can obtain a copy of the License at
-  ~ http://forgerock.org/license/CDDLv1.0.html
-  ~ See the License for the specific language governing
-  ~ permission and limitations under the License.
-  ~
-  ~ When distributing Covered Code, include this CDDL
-  ~ Header Notice in each file and include the License file
-  ~ at http://forgerock.org/license/CDDLv1.0.html
-  ~ If applicable, add the following below the CDDL Header,
-  ~ with the fields enclosed by brackets [] replaced by
-  ~ your own identifying information:
-  ~ "Portions Copyrighted [year] [name of copyright owner]"
-  -->
+<!-- ~ DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+     ~
+     ~ Copyright (c) 2011-2013 ForgeRock AS. All Rights Reserved
+     ~ Portions Copyright 2017 Wren Security.
```

Comment:
Shouldn't these copyrights be at the bottom of the license (like elsewhere)?

Reply:
should now be fixed (per my message above).